src/onewire.c

```diff
@@ -45,13 +45,13 @@ static inline uint32_t getpin() {
 }
 
 static inline uint32_t xferbyte(uint32_t thebyte) {
-	for(uint32_t bits=0; bits<8; bits++) {
+	for (uint32_t bits = 0; bits < 8; bits++) {
 		setpin0();
 		BusyWait(TICKS_US(1.5)); // 1.5us
```

Comment:
Comment unnecessary, it's a left-over since the times when there was a constant "12" in there not making it as obvious.. :)